packages/component/src/components/page-list/mobile.tsx

```diff
@@ -68,6 +70,11 @@ export const AllPageListMobileView = ({
   createNewEdgeless: () => void;
   importFile: () => void;
 }) => {
+  const virtualTableHeight = useVirtualTableHeight(
+    '.main-container',
```

Comment:
Shouldn't use css selector here. We might have multiple page list in a single web